bm/charter_condition.go

```diff
@@ -0,0 +1,47 @@
+package bm
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"time"
+)
+
+// CharterCondition response description
+type CharterCondition struct {
+	Period          *Period   `json:"period,omitempty"`
+	CheckInDays     []*int    `json:"checkInDays,omitempty"`
+	CheckOutDays    []*int    `json:"checkOutDays,omitempty"`
+	MinimumDuration int       `json:"minimumDuration,omitempty"`
+	Overnight       bool      `json:"overnight,omitempty"`
+	CheckInTime     time.Time `json:"defaultCheckInTime,omitempty"`
+	CheckOutTime    time.Time `json:"defaultCheckOutTime,omitempty"`
+	LicenseRequired bool      `json:"licenseRequired,omitempty"`
+	SkipperRequired bool      `json:"requiredSkipperLicense,omitempty"`
+}
```

Comment:
This would not be here. The struct we send back as the response, should not be part of the adapter.